packages-native/listview-swipe/src/ListViewSwipe.xml

```diff
@@ -21,12 +21,12 @@
                 </property>
                 <property key="leftRenderMode" type="enumeration" defaultValue="disabled">
                     <caption>Pattern</caption>
-                    <description>"Show options" will show the background buttons. "Toggle" animates the item back to the initial state. "Swipe out and reset" animates the item out of the list.</description>
+                    <description>"Disabled" ensures your list view items will have no animation or action. "Swipe to show options" enables a partial swipe ?? to show SOMETHING mentioning buttons. "Full swipe to collapse" enables a full swipe to one side which collapses the list item. "Full swipe to reset" enables a full swipe to one side which SOMETHING. "Toggle" enables a partial swipe which does SOMETHING. </description>
                     <enumerationValues>
                         <enumerationValue key="disabled">Disabled</enumerationValue>
-                        <enumerationValue key="buttons">Show options (buttons)</enumerationValue>
-                        <enumerationValue key="archive">Swipe out and collapse</enumerationValue>
-                        <enumerationValue key="swipeOutReset">Swipe out and reset</enumerationValue>
+                        <enumerationValue key="buttons">Swipe to Show options</enumerationValue>
+                        <enumerationValue key="archive">Full swipe to collapse</enumerationValue>
+                        <enumerationValue key="swipeOutReset">Full swipe to reset</enumerationValue>
```

Comment:
Can you please add the same patterns for the Right side?

Reply:
Reached out to you on Slack about this. I will merge as soon as we resolve that discussion.

Reply:
@diego-antonelli Ok I've updated both sides with text Mehmet helped me write. Thank you both. I think this is complete now. Look ok?